pkg/domain/entities/images.go

```diff
@@ -306,6 +306,18 @@ type ImageSaveOptions struct {
 	Quiet bool
 }
 
+// ImageScpOptions provide options for securely copying images to podman remote
+type ImageScpOptions struct {
+	// ImageName is used when loading from remote host
+	ImageName string
+	// ToFrom specifies whether we are copying to or from remote host
+	ToFrom bool
```

Comment:
There _must_ be a way to name this better, ideally so that the field’s meaning is clear even without the comment.

This one isn’t clear even _with_ the comment: “whether we are copying to or from remote host” doesn’t say which one is true and which one is false.

(Using an enum with named values might make this easier — or it might be overkill.)